apps/callscreen/js/call_screen.js

```diff
@@ -461,6 +461,10 @@ var CallScreen = {
     this.mergeButton.classList.add('hide');
   },
 
+  showOnHoldAndMergeContainer: function cs_showOnHoldAndMergeContainer() {
+    this.holdAndMergeContainer.style.display = 'visible';
```

Comment:
This is not a valid value. The fact that it works is probably because it's falling back to the default, which is ```block``` in this case.